lib/api_repo.js

```diff
@@ -77,7 +77,7 @@ function ApiRepo(opts) {
   this.pkgPrefix = opts.pkgPrefix;
   this.overridePlugins = opts.overridePlugins;
   this.zipUrl = opts.zipUrl;
-  this.isGoogleApi = !!opts.isGoogleApi;
+  this.isGoogleApi = !!opts.buildCommonProtos;
```

Comment:
Can we remove `this.isGoogleAPi` itself?  This line looks confusing.

Or renaming `this.isGoogleApi` with `this.buildCommonProtos`?